scripts/templates/diag_table_template

```diff
@@ -58,6 +62,9 @@
  "ocean_model", "fprec",     "fprec",                    "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2
  "ocean_model", "LwLatSens", "LwLatSens",                "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2
  "ocean_model", "Heat_PmE",  "Heat_PmE",                 "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2
+ "ocean_model", "hfds",      "hfds",                     "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2
```

Comment:
I'm not opposed to adding these two variables, but do we have all the parts of these variables (hfds and PRCmE) already? So that it would just be a sum of those to get the net values?

Reply:
If you don't know already, I could also just look that up myself.

Reply:
@JessicaMeixner-NOAA added umo in diag_table. When I make test plots using umo/vmo, I will come by.

Reply:
@JessicaMeixner-NOAA go ahead to check the stuff hfds and PRCmE. I just simply put them there by referring what the tool is using.

Reply:
@JessicaMeixner-NOAA we can undo adding hfds and PRCmE in the table at this branch. We can revisit this issue when I make the plotting script for these. We can undo this change and let @flampouris merge the branch. What do you think?

Reply:
@jkbk2004 sorry I haven't had time to look up if we have all the components for these or not.  I say leave them in for now and let @flampouris merge the branch, and we can always go back and try to remove them by adding the fields up later.

Reply:
Does this change will break anything? If not, I will merge it.

Reply:
No it does not break anything.  It just adds extra fields to the output file that might be a sum of fields that we're already outputting so in the future we might be able to just sum those instead of adding an extra field.

Reply:
:+1: